documentation/src/main/asciidoc/topics/ref_passivation_behavior.adoc

```diff
@@ -0,0 +1,61 @@
+[id='passivation_behavior']
+= Passivation Effects on Cache Loaders
+
+.Passivation disabled
+Writes to data in memory result in writes to persistent storage via the cache
+loader.
```

Comment:
Data is only passivated if a cache store (writer) is configured.

Reply:
@ryanemerson don't we cover this in the section just above? "... cache stores must implement both CacheWriter and CacheLoader interfaces so they can write and load entries from persistent storage."

Reply:
@oraNod Yes, but writes don't happen via "the cache loader" as it is a write, not a load operation.